docs/profiler.md

```diff
@@ -0,0 +1,112 @@
+# What is the profiler tool
+
+The major goal of profiler tool is to provide an estimation how fast can be the proof created.
+Creating a proof in spacemesh depends on few major factors:
+* CPU speed
+* Disk speed
+* Amount of storage initialized
+
+The major task of the profiler tool is to estimate how fast can be the proof created for a given CPU speed and disk speed.
+
+The best way to get the profiler tool is to get it from the [releases page](https://github.com/spacemeshos/post-rs/releases).
+
+It's very important to remember that the proof time itself does not need to be optimized against, we just need to generate valid proof in time which on the mainnet is 12h.
+
+## How to run profiler tool
+
+One of the most important part is to set correct path to data file using `--data-file` flag. It's important to set it so the benchmark will use the proper disk(s). Plaese be warned that the contents of that file MIGHT be overwritten.
+
+By default profiler uses 1GiB sample data you can change that using `--data-size` flag. One can experiment with different sizes to check for the bottlenecks in the hardware.
```

Comment:
```suggestion
By default, profiler uses 1GiB sample data but it can be changed using `--data-size` flag. One can experiment with different sizes to check for bottlenecks in the hardware.
```